### PR TITLE
Component/navbar

### PR DESCRIPTION
**Pull request by karam1ashqar**

### CODE REVIEW

File: client/src/components/header/header.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    -webkit-tap-highlight-color: rgba(0,0,0,0);
+    font-size: 100%;
+}
+
+html, body {
+    margin: 0;
+    overflow:auto;
+    overflow: initial;
+    scroll-behavior: smooth;
+    font-family: 'Inconsolata', monospace;
```
Comment (tamerNasser) [inline client/src/components/header/header.css]: Use the fonts that we decided to use when we made the prototype.

File: client/src/components/header/header.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    render() {
+
+    return ( 
+        <div className="nav">
+        <div className="logo">
+           <a href="#">
+           <p>Save a Life</p>
+           </a>
+        </div>
+<ul className={this.state.slideClass}>
```
Comment (shiryz) [inline client/src/components/header/header.js]: Please align it better

File: client/src/components/header/header.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ <li style={{animation: (`${this.state.animate} 1.0s`)}}><a href="#">Login</a></li>
+ <li style={{animation: (`${this.state.animate} 1.2s`)}}><a href="#">Inspiration</a></li>
+ <li style={{animation: (`${this.state.animate} 1.4s`)}}><a href="#">Donate</a></li>
+ <li style={{animation: (`${this.state.animate} 1.6s`)}}><a href="#">Request</a></li>
+ <li style={{animation: (`${this.state.animate} 1.8s`)}}><a href="#">About</a></li>
+
+</ul>
+<div className={this.state.burger} onClick={this.ToggleSlide}>
+     <div className="line1"></div>
```
Comment (shiryz) [inline client/src/components/header/header.js]: Classes need better names than line1, line2...

File: client/src/components/header/header.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import './header.css';
+
+
+export class Header extends React.Component {
+    state = {
+        slideClass: "nav-links",
+        animate: "",
+        burger: "burger"
+    }
+    ToggleSlide = () => {
```
Comment (shiryz) [inline client/src/components/header/header.js]: I haven't seen all your code but there could be a way to refactor this

File: client/src/components/header/header.js (hunk 3 above)
Comment (karam1ashqar) [reply client/src/components/header/header.js]: What do you suggest calling a line if not line?

File: client/src/components/header/header.js (hunk 4 above)
Comment (karam1ashqar) [reply client/src/components/header/header.js]: Please provide more details and link the wanted-to-refactor lines

**Pull request merged**